packages/entity-list/src/components/ColumnPicker/ColumnPicker.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types'
+import React, {useState} from 'react'
+import {Button, SearchBox, Typography} from 'tocco-ui'
+import styled from 'styled-components'
+import {intlShape} from 'react-intl'
+
+const StyledUl = styled('ul')`
+  && {
+    list-style-type: none;
+  }
+`
+
+const StyledCheckbox = styled('input')`
+  && {
+    vertical-align: top;
+  }
+`
+
+const ColumnPicker = ({columns, onOk, intl}) => {
+  const [selectedColumns, setSelectedColumns] = useState({})
+  const [searchTerm, setSearchTerm] = useState(null)
+  const items = columns
+    .filter(column => searchTerm === null || column.label.match(new RegExp(`${searchTerm}`, 'i')))
+    .map(column => <Typography.Li key={column.id}>
+      <StyledCheckbox
+        type={'checkbox'}
+        id={column.id}
+        checked={Object.prototype.hasOwnProperty.call(selectedColumns, column.id)
+          ? selectedColumns[column.id] : !column.hidden}
+        onChange={value => setSelectedColumns({...selectedColumns, [column.id]: value.target.checked})}/>
+      <Typography.Label for={column.id}>{column.label}</Typography.Label>
+    </Typography.Li>)
+  return <React.Fragment>
+    <SearchBox placeholder={intl.formatMessage({id: 'client.entity-list.preferences.columns.search'})}
+      onSearch={setSearchTerm}/>
+    <StyledUl>{items}</StyledUl>
+    <Button onClick={() => onOk(selectedColumns)}>
+      {intl.formatMessage({id: 'client.entity-list.preferences.columns.button'})}
```

Comment:
i would name it client.entity-list.preferences.columns.okButton to clear things up in the text file (sorry havent thought of that when reviewing the nice2 change)